.changeset/pretty-trees-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mysten/sui.js": patch
```

Comment:
IMO this is actually `minor`, because it's _really_ a major breaking change but we're pre-major so it's minor.